tasks/docs.rb

```diff
@@ -250,6 +236,38 @@ def special_properties(name, data)
       properties
     end
 
+    # Breaks a block of text into the different sections expected for the description,
+    # using the markers "Note:" for "note" sections and "Warning:" for "warning" sections.
+    # TODO: has the limitation that the plain description section is assumed to come first,
+    # and is followed by one or more "note"s or "warning"s sections.
```

Comment:
It probably wouldn't be too difficult to fix this TODO if we wanted to adopt a convention of a blank line in between sections being the end of a section or something like that.